Src/BootCamp.Chapter/Shop.cs

```diff
@@ -64,6 +73,14 @@ public decimal Buy(Item item)
         /// </returns>
         public Item Sell(string item)
         {
+            for (int i = 0; i < Items.Count; i++)
```

Comment:
Don't forget to amend this to use inventory items :)

Reply:
done